apps/frontend/project.json

```diff
@@ -92,20 +93,8 @@
       "command": "gh-pages -d dist/apps/frontend",
```

Comment:
let's remove the deploy command, since we are using CI to deploy, manual deploy can actually break https://github.com/frzyc/genshin-optimizer/pull/1163 in the foreseeable future